package.json

```
@@ -79,14 +82,27 @@
     "node-fetch": "^2.6.7",
     "nth-check": "^2.0.1",
     "parse-link-header": "^2.0.0",
+    "path-to-regexp@0.1.7": "^1.9.0",
+    "qs": "^6.10.3",
     "raw-loader/loader-utils": "^2.0.4",
     "shelljs": "^0.8.5",
     "ssh2": "^1.4.0",
     "tar": "^6.1.9",
     "terser": "^5.14.2",
     "trim": "^0.0.3",
     "trim-newlines": "^3.0.1",
-    "trim-off-newlines": "^1.0.3"
+    "trim-off-newlines": "^1.0.3",
+    "vite@^5.0.0": "^5.0.0",
+    "webpack-dev-middleware": "^5.3.4",
+    "ws@8.13.0": "^8.17.1",
+    "ws@8.16.0": "^8.17.1",
+    "ws@^6.1.0": "^6.2.3",
+    "ws@^7.0.0": "^7.5.10",
+    "ws@^7.2.0": "^7.5.10",
+    "ws@^8.2.3": "^8.17.1",
+    "ws@^8.4.2": "^8.17.1",
+    "ws@^8.5.0": "^8.17.1",
+    "ws@^8.8.0": "^8.17.1"
```

was not able just to set a range*

---

Shouldn't something like this work for the entire `ws@*` dependencies?
```suggestion
    "ws": "^6.2.3 || ^7.5.10 || ^8.17.1"
```

---

unfortunately no, have you had a chance to give it a try?